stories/assets/scss/_mixins.scss

```diff
@@ -8,40 +8,40 @@
 }
 
 @mixin feature__card_bg {
-  background: transparent linear-gradient( 207deg, hsla(0, 0%, 100%, 0.2) 30%, rgba(255, 235, 0, 0.5) 80%, rgba(255, 235, 0, 0.8) ) 0 0 no-repeat padding-box;
+  background: transparent linear-gradient(207deg, hsla(0, 0%, 100%, 0.2) 30%, rgba(255, 235, 0, 0.5) 80%, rgba(255, 235, 0, 0.8)) 0 0 no-repeat padding-box;
 }
 
 //loader animation
 @mixin loader {
   animation: loader-animation 2s linear infinite;
-  -webkit-animation: loader-animation 2s linear infinite;
-  -moz-animation: loader-animation 2s linear infinite;
-  -ms-animation: loader-animation 2s linear infinite;
-  -o-animation: loader-animation 2s linear infinite;
+  animation: loader-animation 2s linear infinite;
+  animation: loader-animation 2s linear infinite;
+  animation: loader-animation 2s linear infinite;
+  animation: loader-animation 2s linear infinite;
 }
 
 @keyframes loader-animation {
-  0% { -webkit-transform: rotate(0deg); }
-  100% { -webkit-transform: rotate(360deg); }
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
 }
 //loader animation end
 
 //transition
 @mixin transition($transitions...) {
-  -webkit-transition: $transitions;
-  -moz-transition: $transitions;
-  -ms-transition: $transitions;
-  -o-transition: $transitions;
+  transition: $transitions;
+  transition: $transitions;
+  transition: $transitions;
+  transition: $transitions;
   transition: $transitions;
 }
 
 //transform
 @mixin transform($transforms) {
-  -webkit-transform: $transforms;
-	-moz-transform: $transforms;
-	-ms-transform: $transforms;
-  -o-transition: $transforms;
-	transform: $transforms;
+  transform: $transforms;
+  transform: $transforms;
+  transform: $transforms;
+  transform: $transforms;
+  transition: $transforms;
```

Comment:
@Shubham-Sharma-77 why we have removed the auto prefixers and now what is the need of this mixin

Reply:
@surbhi90 I think just happened with lint remove auto prefixers, I can fix it